state/internal_test.go

```diff
@@ -66,6 +73,9 @@ func (s *internalStateSuite) SetUpTest(c *gc.C) {
 		},
 		MongoInfo:     info,
 		MongoDialOpts: mongotest.DialOpts(),
+		NewPolicy: func(*State) Policy {
```

Comment:
there's something off about supplying the common registry via two paths -- don't suppose we can eliminate one?